### PR TITLE
add options to runner

### PR DESCRIPTION
**Pull request by alibombaye**
ui-tests should now work with concurrency set to 4 when running locally. 
Single thread ui-tests take about 9m
Concurrent thread ui-tests take about 5m

The script to run this concurrent version is ;
`npm run test:ui`
in this version quarantineMode is set. Which will re-run the test if it fails. It will attempt to re-run the test 3 times if it fails. If it fails all 3 times then the test will fail and it will stop running. Ideally we should be running this at the end when we think the ticket is done to make sure everything is working as expected.

If you want to run ui test for a single section or chrome then as before you pass this in the command line args eg;
`npm run test:ui chrome contact-details`
In this mode it will run in a single thread with quarantineMode off and it will continue to run even when there is a failure. We should use this as we are working in a particular section

The pipeline will use the original script
`npm run test:integration`
This will run in a single thread with quarantine and stopOnFirstFail off.

The runner options will need some tweaking to find the optimal time outs and concurrency. But it'll do for now.

### CODE REVIEW

File: testcafeRunner.js [hunk 1]
```diff
@@ -1,34 +1,50 @@
...
         name: 'nunit',
         output: 'integration-test-report.xml',
       }])
-      .run();
+      .run({
+        selectorTimeout: 3000,
+        assertionTimeout: 1000,
+        pageLoadTimeout: 5000,
+        speed: 1,
+        quarantineMode,
```
Comment (kat1906) [inline testcafeRunner.js]: is it ok to just be putting `true`/`false` here rather than `{ key: value }`?

File: testcafeRunner.js [hunk 2]
```diff
@@ -1,34 +1,50 @@
...
 import routes from './app/routes';
+import { env } from './app/config';
 
 let testcafe;
-let server;
+const app = new App().createApp();
+app.use('/', routes);
+const server = app.listen('1234');
 
 const browserFromArgs = process.argv.slice(2, 3);
```
Comment (kat1906) [inline testcafeRunner.js]: you could put this inside a `if(env === 'test')` block

File: package.json [hunk 3]
```diff
@@ -83,4 +84,4 @@
       "pre-push": "npm run lint"
```
[the remark this answers is not in the document]
Comment (alibombaye) [reply package.json]: I don't really trust it enough yet. Let's see how stable it is and then it'll be worth adding it there perhaps 🤷‍♂

File: testcafeRunner.js (hunk 1 above)
Comment (alibombaye) [reply testcafeRunner.js]: Not sure I get what you mean

Comment (kat1906) [reply testcafeRunner.js]: never mind I am being stupid 🔨

**Pull request merged**